Source/ThirdParty/libwebrtc/CMakeLists.txt

```diff
@@ -1880,9 +1888,22 @@ if (WTF_CPU_ARM64)
         Source/webrtc/common_audio/third_party/ooura/fft_size_128/ooura_fft_neon.cc
         Source/webrtc/modules/audio_processing/aecm/aecm_core_neon.cc
     )
-    add_definitions(-DWEBRTC_ARCH_ARM64)
+    list(APPEND webrtc_SOURCES ${webrtc_neon_SOURCES})
     add_definitions(-DWEBRTC_HAS_NEON)
-endif()
+endif ()
+
+if (WTF_CPU_ARM)
+    list(APPEND webrtc_neon_SOURCES
+        Source/third_party/pffft/src/pffft.c
+        Source/webrtc/modules/audio_processing/aec3/adaptive_fir_filter.cc
+        Source/webrtc/modules/audio_processing/aec3/adaptive_fir_filter_erl.cc
+        Source/webrtc/modules/audio_processing/aec3/comfort_noise_generator.cc
+        Source/webrtc/modules/audio_processing/aec3/matched_filter.cc
+        Source/webrtc/modules/audio_processing/aec3/suppression_filter.cc
+        Source/webrtc/modules/audio_processing/aec3/suppression_gain.cc
```

Comment:
Why are we building those only on ARM? looking at pfft it seems like it should build on x86 too.

Reply:
These files are also added for x86 too. See:

https://github.com/WebKit/WebKit/blob/main/Source/ThirdParty/libwebrtc/CMakeLists.txt#L582
https://github.com/WebKit/WebKit/blob/main/Source/ThirdParty/libwebrtc/CMakeLists.txt#L1090

What this snippet of code does is to define a new group of files (`webrtc_neon_SOURCES`) and then apply the build flags `-mfpu=neon;-mfloat-abi=hard`. This is only done if the build target is ARM 32-bit (maybe I should change conditional `WTF_CPU_ARM` for `WTF_CPU_ARM_THUMB2`).